lib/renderer/window-setup.ts

```diff
@@ -20,8 +20,9 @@ import { ipcRendererInternal } from '@electron/internal/renderer/ipc-renderer-in
 const { defineProperty } = Object
 
 // Helper function to resolve relative url.
-const a = window.document.createElement('a')
+let a: HTMLAnchorElement
 const resolveURL = function (url: string) {
+  a = a || window.document.createElement('a')
```

Comment:
can we just use the [`URL` API](https://developer.mozilla.org/en-US/docs/Web/API/URL)?

Reply:
Unknown 🤷‍♂